test/discr/discr_ops_test.py

```diff
@@ -0,0 +1,218 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Unit tests for `discr_ops`."""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+
+import pytest
+import numpy as np
+
+import odl
+from odl.discr.discr_ops import _SUPPORTED_RESIZE_PAD_MODES
+from odl.util.testutils import almost_equal, example_element
+
+
+# --- ResizingOperator --- #
+
+
+def test_resizing_op_init():
```

Comment:
Add the `fn_impl` fixture and add `impl=fn_impl` to the spaces.

You may also want to test going from one impl to another, it is not clear if this is supported or not.